OptimizelySDK/ProjectConfig.cs

```diff
@@ -177,13 +212,18 @@ private void Initialize()
             Events = Events ?? new Entity.Event[0];
             Attributes = Attributes ?? new Attribute[0];
             Audiences = Audiences ?? new Audience[0];
+            FeatureFlags = FeatureFlags ?? new FeatureFlag[0];
+            FeatureVariables = FeatureVariables ?? new FeatureVariable[0];
+            Rollouts = Rollouts ?? new Rollout[0];
 
             _ForcedVariationMap = new Dictionary<string, Dictionary<string, string>>();
             _GroupIdMap = ConfigParser<Group>.GenerateMap(entities: Groups, getKey: g => g.Id.ToString(), clone: true);
             _ExperimentKeyMap = ConfigParser<Experiment>.GenerateMap(entities: Experiments, getKey: e => e.Key, clone: true);
             _EventKeyMap = ConfigParser<Entity.Event>.GenerateMap(entities: Events, getKey: e => e.Key, clone: true);
             _AttributeKeyMap = ConfigParser<Attribute>.GenerateMap(entities: Attributes, getKey: a => a.Key, clone: true);
             _AudienceIdMap = ConfigParser<Audience>.GenerateMap(entities: Audiences, getKey: a => a.Id.ToString(), clone: true);
+            FeatureKeyMap = ConfigParser<FeatureFlag>.GenerateMap(entities: FeatureFlags, getKey: f => f.Key, clone: true);
```

Comment:
why assigning directly to properties, plz. set in private variable and return private variable from public getter property as the trend we have in C#.

Reply:
I think the Properties approach is better then private variable. Thoughts?